purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -119,7 +128,22 @@ internal class BillingWrapper(
             billingClient?.let {
                 if (!it.isReady) {
                     log(LogIntent.DEBUG, BillingStrings.BILLING_CLIENT_STARTING.format(it))
-                    it.startConnection(this)
+                    if (startConnectionAttempts < MAX_RECONNECTION_ATTEMPTS) {
+                        startConnectionAttempts++
+                        try {
```

Comment:
Thought of adding this just in case, but maybe we should try the fix without the try/catch and make a release? I am not sure

Reply:
Well, I think it might be worth adding it to prevent any crashes as mentioned in the other comment